### DISCUSSION TITLE
Can we get hash partially matched?

### DISCUSSION THREAD

**Discussion by swhoro** [ScoopInstaller/Scoop#5878] [Help and Questions]
#### Is your feature request related to a problem? Please describe.

For example, the md5 of a file is `897bf087643818e9f797d48212f226f6`
But on its website, I cannot get string `897bf087643818e9f797d48212f226f6` but only `897bf087`
In this situation, how should I write `autoupdate.hash` field in manifest.json.

**Comment by rashil2000** [comment]
Although collisions are rare and this can be supported technically, I would still not advise doing this: the manifest should get the entire hash for safety reasons.
